next.config.js

```diff
@@ -21,6 +21,42 @@ module.exports = {
       }
     ]
   },
+  async rewrites() {
+    return [
+      {
+        source: '/validators',
+        destination: '/'
+      },
+      {
+        source: '/builders',
+        destination: '/'
+      },
```

Comment:
OK, but do we want to have dedicated `/validators` and `/builders` routes ? 

They have no entry point, they are useless.

IMHO, in the header, when you click on `Validators` or `Builders`, it should not push a new route in history, but only open the submenu as side effect.